src/app/locales/fr/modal.yml

```diff
@@ -46,9 +46,9 @@ quit:
 streaming:
     header:
         preparing: "Préparation, veuillez patienter"
-        launching: "Lancement: "
-        watching: "En visionnage: "
-        ended: "Le stream est terminé: "
+        launching: "Lancement : "
+        watching: "En visionnage : "
+        ended: "Le stream est terminé : "
```

Comment:
There's not a lot of space available in the headline of the modal dialogs (remember that the channel name which comes afterwards does have a variable length), so even if this is grammatically correct, I'd prefer not having spaces here to avoid potential issues with the length of the text.